app/src/main/java/ru/netology/fmhandroid/ui/FeedFragment.kt

```diff
@@ -0,0 +1,31 @@
+package ru.netology.fmhandroid.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import ru.netology.fmhandroid.databinding.FragmentFeedBinding
+
+class FeedFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+
+        val binding = FragmentFeedBinding.inflate(
+            inflater,
+            container,
+            false
+        )
+
+        return binding.root
+    }
```

Comment:
Подправлять не обязательно, но можешь взять на карандаш следующий вариант:
```suggestion
    ) = FragmentFeedBinding.inflate(
        inflater,
        container,
        false
    ).root
```
Если появится код конфигурации, то всегда можно добавить `apply` или `also`:
```suggestion
    ) = FragmentFeedBinding.inflate(
        inflater,
        container,
        false
    ).also { binding ->
        ...
    }.root
```

Reply:
Круто :) Записал себе в заметки этот вариант